apps/web/components/Embed.tsx

```diff
@@ -726,6 +769,274 @@ const ChooseEmbedTypesDialogContent = () => {
   );
 };
 
+const EmailEmbed = ({ eventType, username }: { eventType?: any; username: string }) => {
+  const { t } = useLocale();
+
+  const [timezone] = useTimePreferences((state) => [state.timezone]);
+
+  const [selectTime, setSelectTime] = useState(false);
+
+  useInitializeBookerStore({
+    username,
+    eventSlug: eventType?.slug ?? "",
+    eventId: eventType?.id,
+    layout: BookerLayouts.MONTH_VIEW,
+  });
```

Comment:
Need to initialise the booker store here to access the required state variables from the store in the child components